aws-http-nio-client-netty/pom.xml

```diff
@@ -50,6 +50,8 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-transport-native-epoll</artifactId>
+            <version>${netty.version}</version>
+            <classifier>linux-x86_64</classifier>
```

Comment:
I like this optimization but I don't like that it's going to get pulled in on every architecture.

I think we should default to the Nio implementation and at runtime pickup the Epoll or KQueue implementations if they're there.